translucentsabre-huntsvilleal.md

```diff
@@ -0,0 +1,10 @@
+# Huntsville, AL
```

Comment:
Would you recommend visiting Huntsville?

Thanks for informing me that this is AL!